Sources/SwiftDocC/Model/Rendering/Variants/VariantCollection+Variant.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
 
 public extension VariantCollection {
     /// A variant for a render node value.
-    struct Variant<Value: Codable> {
```

Comment:
What's the reason for this change?

Reply:
`VariantCollection` is already generic and with `Variant` as an inner struct I _think_ the original intention was for `VariantCollection.Variant` to always be generic over the same value as `VariantCollection` but with inner declaration defining its own variant they weren't. All previous places in the code repeated the same value (for example `VariantCollection<TransformedValue>.Variant<TransformedValue>`) but I don't think we intended for a `String` variant collection to have `Int` variants, so I removed it.

I would have left it as-is if it wasn't that the two levels of generic made it harder to specify the `Equatable where Value: Equatable` constraints to add equatable conformance.

Reply:
Got it – thank you!